### ISSUE TITLE
When Sakai sites are created through Kits, add all crosslisted rosters to the site

### ISSUE THREAD

**Issue by joliet** [DukeLearningInnovation/kits#492]
I am an instructor trying to create a Sakai site for my crosslisted course through Kits, but not all of the crosslisted rosters are added to the Sakai site. This makes me confused because I can see all the crosslisted rosters in Kits and don't know why they are not in my Sakai site.

How might we (the Kits project team). automatically add the crosslisted rosters to Sakai sites created through Kits?

**Questions**

**Assumptions**

**Hypotheses**

**Comment by ewendland23** [comment]
Code will now find and add all rosters for every cross-listed course when creating a new Sakai site.

**Comment by ewendland23** [comment]
Need to address the reverse of this.  When coming from a Sakai site with multiple rosters check if they are all for the same, cross-listed course.

**Comment by joliet** [comment]
I tested this both ways - starting in kits-test and creating a Sakai site and then creating a Sakai site in sakai-test and creating a cross-listed course site.  I tested with Laurent Dubois' course AAS 890S 01 (starting in Kits) and Kelly Alexander's course DOCST:344S:01:6319:1710 (starting in Sakai).  The Sakai site created in Kits included all the cross-listed rosters.  The Kit for DOCST 344S 01 Sp20 showed as a cross-listed course and the Sakai site linked to the correct Sakai site.
